src/components/LogbookContext/index.tsx

```diff
@@ -82,6 +89,7 @@ type Context = {
   getRecentLogbooks: (limit?: number) => Promise<void>
   getLogbook: (tokenId: string) => Promise<void>
   logbooks: { [tokenId: string]: Logbook | undefined }
+  recentLogbooks: RecentLogbooks
```

Comment:
@tx0c add `recentLogbooks` to store tokenIds since `logbooks` contain all logbooks not only recents.